pkg/lib/cockpit-components-dropdown.tsx

```diff
@@ -36,7 +35,14 @@ import { EllipsisVIcon } from '@patternfly/react-icons';
  * require a separator between DropdownItem's use PatternFly's Divivder
  * component.
  */
-export const KebabDropdown = ({ dropdownItems, position = "end", isDisabled = false, toggleButtonId, isOpen, setIsOpen, props }) => {
+export const KebabDropdown = ({ dropdownItems, position = "end", isDisabled = false, toggleButtonId, isOpen, setIsOpen, props } : {
+    dropdownItems: React.ReactNode,
+    position?: "start" | "end" | "right" | "left" | "center",
```

Comment:
Why is this nullable? This is an enum/bool with a defined default value, and I don't think we ever want to allow passing an explicit "null" or "undefined"?

Reply:
`?` doesn't mean `null`.  It means "this argument may be omitted".

Reply:
Without the `?` this happens:

![image](https://github.com/cockpit-project/cockpit/assets/36541154/0b0dbba5-23c7-45d6-994c-07669308e83a)

Reply:
ugh -- that means that the default argument is not working at all? Does tsc somehow compile that out? That looks like a very serious regression then. Not sure if we cover this in pixel tests, but this is a huge red flag to me.

Reply:
The default option *is* working.  And it is possible to omit the property with the `?` is present (as it is in the version of the code proposed in the PR).

Reply:
Ah sorry, I misread the screenshot above as a runtime brwoser console error (from propTypes in developmen mode). I suppose it's "just" a tsc error then? But then this is really stupid -- seems that tsc doesn't understand default arguments properly.